app/views/items/show.html.erb

```diff
@@ -1,37 +1,53 @@
-<h2 class="itemname"><%= "Item details for #{@item.name}" %></h2>
-
-<h4>Description</h4>
-<div class="item-desc">
-  <p><%= @item.description %> </p>
-</div>
-<h4>Status</h4>
-<div class="item-status">
-  <p><%= @item.status %></p>
-</div>
-<h4>Image</h4>
-<div class="item-image">
-   <% if @item.image.attached?%> 
-   <%= image_tag(@item.image)%> 
-   <% else %>
-   <%= image_tag('image_placeholder.jpg',size: '300')%> 
-   <% end %>
-</div>
-<h4>Deliverable</h4>
-<div class="item-deliverable">
-  <% if @item.listed %> 
-    <p><%= "yes"%></p>
-  <% else %>
-    <p><%= "no" %></p>
-  <% end %>
-</div>
-<h4>Price</h4>
-<div class="item-price">
-  <p><%= number_to_currency(@item.price) %></p>
-</div>
-
-<h4>Sold by: </h4>
-<p> <%= link_to(@seller , user_path(:userID => @sellerID)) %> </p>
-
-<div>
-  <%= link_to("Back to item list", items_path, :class => "btn btn-primary") %>
-</div>
\ No newline at end of file
+<body>
+    <div class="lightbox-blanket">
+        <div class="pop-up-container">
+           <div class="pop-up-container-vertical">
+                <div class="pop-up-wrapper">
+                   <div class="go-back">
+
+                    <%= link_to("Back", items_path) %>
+
+                    </div>
+                    <div class="itemlist">
+                        <div class="itemlist-item-wrapper">
```

Comment:
if you're trying to apply multiple classes at once, I would stay away from creating lots of divs. Instead, you can say `<div class="itemlist-item-wrapper product-details product-left ....etc"`

If that wasn't your intention, let me know.

Reply:
+1 on ^

Reply:
you can have a lot of classes for class= classname1 classname2 classname3, etc.